modules/react-components/src/hooks/use-ui-element-sizes.tsx

```diff
@@ -63,5 +87,9 @@ export const useUIElementSizes = (): ElementSizesForUIPropsInterface => {
     // This method has to tracked in both identity apps and extensions to fix the eslint issues,
     // hence it can be fixed later.
     // Tracked here: https://github.com/wso2/product-is/issues/12726#issuecomment-954835957
-    return { headerHeight, footerHeight };
+    return {
+        headerHeight,
+        // eslint-disable-next-line sort-keys
```

Comment:
Any reason to disable sort keys here?

Reply:
A developer had added this comment.
```
    // This method has to tracked in both identity apps and extensions to fix the eslint issues,
    // hence it can be fixed later.
    // Tracked here: https://github.com/wso2/product-is/issues/12726#issuecomment-954835957
```

I will check the validity of this in a follow up PR.